.travis.yml

```diff
@@ -11,13 +11,11 @@ env:
 matrix:
   fast_finish: true
   include:
-    - python: 3.5
-      env: TOX_ENV=py35-tests
-    - python: 3.5
-      env: TOX_ENV=py35-flake8
-    - python: 2.7
-      env: TOX_ENV=py27-flake8
-    - python: 2.7
+    - python: 3.6
+      env: TOX_ENV=tests
+    - python: 3.6
+      env: TOX_ENV=black
+    - python: 3.6
```

Comment:
Seems like the first should run on 3.6, 3.7, 3.8 and the rest on 3.8, no?

Reply:
I've moved this to https://github.com/twisted/incremental/pull/64